docs/cli-spec.md

```diff
@@ -0,0 +1,40 @@
+## Polystat usages
+
+### Notation
+`|` - exclusive or
+`(...)` - must be present
+`[...]` - may not be present
+`<thing>...` -- multiple instances of thing can be accepted.
+```
+polystat (eo | java | python) --tmp <path> --in <path> [--include <rule>... | --exlude <rule>...] --sarif  --files [<path>]
+polystat --version --help
+polystat list
+```
+## Input configuration
+* The subcommand specifies which files should be analyzed (`.eo`, `.java` or `.py`). More languages can be added in the future. 
+* Analyzes the input files in the `--in` directory. If `--in` is not specified, defaults to reading the code in the input language from stdin. 
+* `--in` can also accept a path which leads to a file. In this case, only the specified file will be analyzed. 
+* The temporary files produced by analyzers are available in `--tmp` directory.  If `--tmp` is not specified, temporary files are stored in the OS-created tempdir. Each target language may have a different structure of the files in the temporary directory.
```

Comment:
1. This implies `[--tmp <path>]`. 
2. Will `--tmp <path>` write to several directories?
Overall, I suppose, the flag should be `[--tmp path]`

Reply:
@br4ch1st0chr0n3 I rewrote the description using [this](https://en.wikipedia.org/wiki/Command-line_interface#:~:text=Command%20description%20syntax%5Bedit%5D) syntax guide.  What do you think?